companion.py

```diff
@@ -565,6 +566,9 @@ def query(self, endpoint: str) -> CAPIData:  # noqa: CCR001
 
         try:
             logger.trace_if('capi.query', 'Trying...')
+            if conf_module.capi_pretend_down:
+                raise ServerError(f'Pretending CAPI is down for {endpoint} endpoint')
```

Comment:
Change this to

`raise ServerConnectionError(f'Pretending CAPI down: {endpoint}')`